website/docs/source/v2/provisioning/salt.html.md

```diff
@@ -65,6 +65,7 @@ you can specify a branch, tag, or any treeish. Not supported on Windows.
 * `always_install`   (boolean) - Installs salt binaries even
  if they are already detected, default `false`
 
+* `bootstrap_script` (string) - Path to your customized salt-bootstrap.sh script.
```

Comment:
Hi @michaeldhopkins 

Have you had a chance to rebase this yet? Please let me know if you have any questions.